euth/dashboard/forms.py

```diff
@@ -282,18 +288,20 @@ def save(self, commit=True):
                 if not delete:
                     self._create_phase(phase, commit, module)
 
-            if (phase['end_date'] and
-                    phase['end_date'].replace(tzinfo=None) >
-                    datetime.datetime.utcnow()):
-                no_phase_left = False
-
-        project.is_archived = project.is_archived and no_phase_left
-
         if commit:
             project.save()
             if 'module_settings' in objects:
                 objects['module_settings'].save()
 
+    def clean(self):
```

Comment:
Done in https://github.com/liqd/a4-opin/commit/a44b1c2704557bc776aff76081e8c4cca0e2d904.